tests/usb/usb.go

```diff
@@ -103,6 +106,22 @@ var _ = Describe("[Serial][sig-compute][USB] host USB Passthrough", Serial, deco
 				&expect.BSnd{S: fmt.Sprintf("%s\n", cmdNumberUSBs)},
 				&expect.BExp{R: console.RetValue(fmt.Sprintf("%d", len(deviceNames)))},
 			}, 15)).To(Succeed(), "Device not found")
+
+			By("Verifying ownership is properly set in the host")
+			domainXml, err := tests.GetRunningVMIDomainSpec(vmi)
+			Expect(err).ToNot(HaveOccurred())
+
+			for _, hostDevice := range domainXml.Devices.HostDevices {
+				if hostDevice.Type != api.HostDeviceUSB {
+					continue
+				}
+				addr := hostDevice.Source.Address
+				path := fmt.Sprintf("%sdev/bus/usb/00%s/00%s", pkgUtil.HostRootMount, addr.Bus, addr.Device)
+				cmd := []string{"stat", "-c", `"%u %g"`, path}
+				stdout, err := tests.ExecuteCommandInVirtHandlerPod(nodeName, cmd)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(stdout).Should(ContainSubstring(fmt.Sprintf("%d %d", pkgUtil.NonRootUID, pkgUtil.NonRootUID)))
```

Comment:
Let's hardcore these values so we see failure if we ever change them.

Reply:
Ok, will do!